reference/mongodb/bson/decimal128/construct.xml

```diff
@@ -11,7 +11,7 @@
   &reftitle.description;
   <methodsynopsis>
    <modifier>final</modifier> <modifier>public</modifier> <methodname>MongoDB\BSON\Decimal128::__construct</methodname>
-   <methodparam choice="opt"><type>string</type><parameter>value</parameter></methodparam>
+   <methodparam><type>string</type><parameter>value</parameter></methodparam>
```

Comment:
Noted that this relates to https://github.com/mongodb/mongo-php-driver/pull/1347#discussion_r948261399